psydac/api/fem.py

```diff
@@ -655,10 +669,19 @@ def allocate_matrices(self, backend=None):
             self._matrix = BlockLinearOperator(trial_space, test_space)
 
         if is_broken:
-            i, j = self.get_space_indices_from_target(domain, target)
+            i,j = self.get_space_indices_from_target(domain, target )
```

Comment:
There is an extra space after `target`

Reply:
Fixed in [6b86a05](https://github.com/pyccel/psydac/pull/241/commits/6b86a05a0bd8c6b5c3f62695c8f330b8ac656633)